src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -213,21 +214,24 @@ public partial class JsonException : System.Exception
     {
         public JsonException() { }
         protected JsonException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
-        public JsonException(string message) { }
-        public JsonException(string message, System.Exception innerException) { }
-        public JsonException(string message, string path, long? lineNumber, long? bytePositionInLine) { }
-        public JsonException(string message, string path, long? lineNumber, long? bytePositionInLine, System.Exception innerException) { }
+        public JsonException(string? message) { }
+        public JsonException(string? message, System.Exception? innerException) { }
+        public JsonException(string? message, string? path, long? lineNumber, long? bytePositionInLine) { }
+        public JsonException(string? message, string? path, long? lineNumber, long? bytePositionInLine, System.Exception? innerException) { }
         public long? BytePositionInLine { get { throw null; } }
         public long? LineNumber { get { throw null; } }
-        public override string Message { get { throw null; } }
-        public string Path { get { throw null; } }
+#pragma warning disable 8609
+        public override string? Message { get { throw null; } }
+#pragma warning restore 8609
+        public string? Path { get { throw null; } }
```

Comment:
> should we make the path in the ctor be non-nullable, and then make this non-nullable, too?

Changing the (string) and (string, Exception) ctors to assign "" instead of null?

Reply:
So am gonna change it non null and set empty string wherever it setting null. There is also assert checking it for null but seems useless will delete it